src/app/[locale]/(pages)/esci/conferma/page.tsx

```diff
@@ -1,12 +1,47 @@
 'use client';
 
 import { useTranslations } from 'next-intl';
-import SessionExistingComp from '@/app/[locale]/_component/sessionExistingComp/sessionExistingComp';
+import { useState, useEffect } from 'react';
+import SessionActiveComp from '@/app/[locale]/_component/sessionActiveComp/sessionActiveComp';
+import { WebProfileApi } from '@/api/webProfileApiClient';
+import NoSessionActiveComp from '@/app/[locale]/_component/noSessionActiveComp/noSessionActiveComp';
+import { storageUserOps } from '@/app/[locale]/_utils/storage';
+import { SessionState } from '@/api/generated/webProfile/SessionState';
 
-const Session = (): React.ReactElement => {
+const LogoutConfirm = (): React.ReactElement => {
+  const [sessionData, setSessionData] = useState<SessionState>();
   const t = useTranslations('ioesco');
+  const userFromStorage = storageUserOps.read();
+  const isL1 = userFromStorage?.spidLevel === process.env.NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1;
 
-  return <SessionExistingComp title={t('profilelogout.logout')} showArrowBackBtn={true} />;
+  useEffect(() => {
+    WebProfileApi.getUserSessionState()
+      .then((res) => {
+        setSessionData(res);
+      })
+      .catch((err) => {
+        // eslint-disable-next-line no-console
+        console.log(err);
+      });
+  }, []);
+
+  const renderSessionActive = (): React.ReactElement => {
+    if (sessionData?.session_info.active) {
```

Comment:
What about the `expiration_date` if it is `expired`?

Reply:
When the date its expired the BE returns in session_info the state active as false. The date logic is already done in the BE. We only have to check sessionData?.session_info.active in the FE.